docs/4.1. Behaviour - Node Start-Up.md

```diff
@@ -0,0 +1,24 @@
+# Behaviour: Node Start-Up
+
+_(c) AMWA 2020, CC Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)_
+
+This document covers expected behaviour for interactions between a Node (the client) and the System API (the server) at start-up.
+
+## Normal Operation
+
+1. The Node performs a DNS-SD browse for services of type '\_nmos-system.\_tcp' as specified by the client interaction procedure in [Discovery: Operation](3.1.%20Discovery%20-%20Operation.md).
+2. The Node fetches the global configuration resource (`/global`).
+3. The Node should use the unique System ID and the version timestamp (`id` and `version`) to determine whether the system environment has changed since its last start-up, for example, by comparing against information held in non-volatile storage.
+4. After successfully fetching the global configuration resource, a Node is not required to periodically repeat this operation until its next restart.
+
+If the Node determines a different System ID is being advertised or the version timestamp is newer, it may not be appropriate for the Node to restore its previous operating state without further checks.
+
+## Error Conditions
```

Comment:
I wonder if we need some extra guidance here on what to do if no system resources at all are found via DNS-SD, or at least none where the TXT records are applicable? If the DNS server responded but with no records it may be safe for the Node to assume there is no System API and not retry, but if the DNS server itself is down then this may warrant similar behaviour to the below.

Reply:
I'm not convinced we can distinguish these cases, esp. as we allow both mDNS and unicast DNS-SD, and an mDNS advertisement might arrive at any point? The mention of exponential backoff without any suggested minimum/maximum/back-off ratio (as in IS-04) allows the Node to choose very large values...

Reply:
Ok, just seems a little non-ideal that a Node then has to maintain a background process (however infrequently it runs) in environments where a System API isn't present. I agree though that if a system is using mDNS only then there is no real equivalent to getting a DNS response with no matching results.

Reply:
Additional sentence added